README.md

```diff
@@ -1 +1,64 @@
-# support-page
\ No newline at end of file
+# Support Page
+
+Use the Support Page Library with the GOV.UK Laravel library to insert a Support Page and Support Admin Page.
```

Comment:
This makes it sound like the support page library is a different thing.

Start with a description of what the support page library is, noting that it is designed for use with the GOV.UK Laravel library.